### PR TITLE
add pg8000

### PR DESCRIPTION
**Pull request by nicolas-geniteau**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by nicolas-geniteau** [comment]
@conda-forge/help-python ready for review, thanks !

File: recipes/pg8000/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2007-2009, Mathieu Fenniak
```
Comment (chrisburr) [inline recipes/pg8000/LICENSE.txt]: It looks like this file is included in the tarball. If so it can be deleted

File: recipes/pg8000/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  sha256: eebcb4176a7e407987e525a07454882f611985e0becb2b73f76efb93bbdc0aab
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - python >3.5
```
Comment (chrisburr) [inline recipes/pg8000/meta.yaml]: The setup.py claims:
```suggestion
    - python >=3.5
```

File: recipes/pg8000/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    - pip
+  run:
+    - python >3.5
```
Comment (chrisburr) [inline recipes/pg8000/meta.yaml]:
```suggestion
    - python >=3.5
```

**Pull request merged**